sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.5.0-beta.3 (Unreleased)
-
+## 1.5.0 (2023-11-07)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- With the new API version `2023-10-01`, the configuration snapshot feature is generally available.
```

Comment:
Since this is a big change, please mention the relevant classes and methods for the feature and link any samples.